apps/opik-backend/src/main/java/com/comet/opik/domain/TraceService.java

```diff
@@ -251,10 +253,12 @@ private Mono<Project> getProjectByName(String projectName) {
     }
 
     private <T> Mono<T> failWithConflict(String error) {
+        log.info(error);
```

Comment:
To avoid pollution, it may have sense to log only once by identifier (projectName) in case these requests may be frequent. Smth like we have in EM: https://github.com/comet-ml/comet-backend/blob/f287bed3cfa92ba502fdb1a5728cd2ca1e55dd9a/comet-utils/src/main/java/com/comet/utils/monitoring/ErrorReporter.java#L57

Reply:
Conflicts are constraint validation. I don't expect to have them very often, but we can consider this if the number increases considerably. I would just avoid adding it now if it is not necessary yet.